src/java.base/windows/native/libnio/ch/DatagramChannelImpl.c

```diff
@@ -89,7 +89,7 @@ Java_sun_nio_ch_DatagramChannelImpl_disconnect0(JNIEnv *env, jclass clazz,
 
     rv = connect((SOCKET)fd, &sa.sa, sa_len);
     if (rv == SOCKET_ERROR) {
-        handleSocketError(env, WSAGetLastError());
+        NET_ThrowNew(env, WSAGetLastError(), "connect");
```

Comment:
Can you check the copyright date on this and the other files as some of these haven't been updated in a while.

Reply:
Hi Alan, I adjusted the copyright dates.